clang/include/clang/Driver/Options.td

```diff
@@ -3844,10 +3844,12 @@ def fdebug_macro : Flag<["-"], "fdebug-macro">, Group<f_Group>,
 def fno_debug_macro : Flag<["-"], "fno-debug-macro">, Group<f_Group>,
   Visibility<[ClangOption, CLOption, DXCOption]>,
   HelpText<"Do not emit macro debug information">;
+def fsystem_debug : Flag<["-"], "fsystem-debug">, Group<f_Group>,
+  Visibility<[ClangOption, CC1Option, CLOption, DXCOption]>,
+  HelpText<"Emit debug information for declarations in system headers">;
 def fno_system_debug : Flag<["-"], "fno-system-debug">, Group<f_Group>,
```

Comment:
Do we still need this option?

Thanks

Reply:
Originally, we only had -fno-system-debug, but now -fno-system-debug is on by default, so -fsystem-debug is important to revert to old behavior.  -fsystem-debug is needed for the testcase linetable-eh.cpp

It could be argued that -fno-system-debug is no longer necessary, since it is on by default.  However, it is better to have on/off pairs for a boolean option.